src/universal/components/ReflectionGroupHeader.js

```diff
@@ -18,20 +17,19 @@ const GroupHeader = styled('div')(({isExpanded, phaseType}) => ({
   display: 'flex',
   fontSize: '.875rem',
   justifyContent: isExpanded ? 'flex-start' : phaseType === VOTE ? 'space-between' : 'center',
-  paddingBottom: 8,
+  padding: '0 .75rem .5rem',
   width: '100%'
 }))
 
 const TitleAndCount = styled('div')(({isExpanded}) => ({
   alignItems: 'center',
   display: 'flex',
   flexShrink: 1,
-  justifyContent: !isExpanded && 'center',
-  position: 'relative'
+  justifyContent: 'space-between',
```

Comment:
this breaks the expanded view by putting the card count on the far right, which looks strange during the closing animation.
repro: expand 3 card stack in a 5 column layout